src/_default-config.scss

```diff
@@ -34,6 +34,11 @@ $gorko-colors: (
 /// to enabling/disabling pre-built components/utilities.
 ///
 $gorko-config: (
+  'namespace': (
+    'prefix': '',
+    'classes': true,
+    'vars': false
+  ),
```

Comment:
Once concern I have here is that we need to support people updating Gorko and not being aware of this feature. Is there an accommodation for this config section not existing?